notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Do not delete the service account. The notebook does not create the service account. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>done</p>